cmake/build_mingw.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ -z "$ARCH" ]; then
+	ARCH=32
```

Comment:
I think `ARCH` should default to 64-bit.  I know zero people in 2024 that prefer 32-bit builds of anything.

Reply:
done